docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,34 @@
+# Deploying the Node Controller Manager into a Kubernetes cluster
+
+As already mentioned, the Node Controller Manager is designed to run as controller in a Kubernetes cluster. The existing source code can be compiled and tested on a local machine as described in [Setting up a local development environment](../development/local_setup.md). Once the implementation is stabilized, you can deploy this onto your cluster using the steps described below.
+
+## Prepare the cluster
+
+- Connect to the remote kubernetes cluster where you plan to deploy the Node Controller Manager using the kubectl. Make the KUBECONFIG environment variable point to the yaml file containing your cluster info.
```

Comment:
s/Make the KUBECONFIG environment variable point to the yaml file containing your cluster info./Set the environment variable KUBECONFIG to the path of the yaml file containing the cluster info./

Reply:
Done.
https://github.com/gardener/node-controller-manager/commit/95c68a4c3f483a5f770c7de61f12b25e5411f3c0#diff-04c6e90faac2675aa89e2176d2eec7d8R12